app/controllers/neighborhoods_controller.rb

```diff
@@ -13,6 +13,7 @@ def show
 
   def new
     @neighborhood = Neighborhood.new
+    @errors = @neighborhood.errors
```

Comment:
Display of errors will need to be cleaned up. I want you to do research into when you have access to the errors, to avoid making this assignment in your controller so often.